src/Gui/DlgParameterImp.cpp

```diff
@@ -646,19 +646,19 @@ ParameterValue::ParameterValue( QWidget * parent )
   : QTreeWidget(parent)
 {
     menuEdit = new QMenu(this);
-    changeAct = menuEdit->addAction(tr("Change value"), this, SLOT(onChangeSelectedItem()));
+    changeAct = menuEdit->addAction(tr("Change value"), this, qOverload<>(&ParameterValue::onChangeSelectedItem));

```

Comment:
Why is there an empty overload?

Reply:
Because the argument of ParameterValue::onChangeSelectedItem is void. Since writing `qOverload<void>...` is not allowed it must be `qOverload<>...`

Reply:
Thanks. Good to know that void is not allowed in this circumstance.